Documentation/sources/content-card-ui/public-classes/uiaction.md

```diff
@@ -0,0 +1,103 @@
+# Sealed Class - UIAction
```

Comment:
Is this supposed to be `UIEvent`, not `UIAction`?

Reply:
yup, my mistake. fixed this up.